.github/actions/constellation_iam_create/action.yml

```
@@ -104,3 +110,10 @@ runs:
           --update-config \
           --tf-log=DEBUG \
           --yes
+
+    - name: Set STACKIT-specific configuration
+      shell: bash
+      run: |
+        yq eval -i "(.provider.openstack.stackitProjectID) = \"6563baab-869d-4e7f-93cc-4087b4165073\"" constellation-conf.yaml
```

```suggestion
        yq eval -i "(.provider.openstack.stackitProjectID) = \"6563baab-869d-4e7f-93cc-4087b4165073\"" constellation-conf.yaml
```
nit: project ID could be set through a secret / action input